src/hu_nmt/data_augmentator/augmentators/subject_object_augmentator.py

```diff
@@ -488,17 +488,17 @@ def dump_augmented_sentences_to_files(self):
         if self.output_format == 'tsv':
             for augmentation_type in augmentation_types:
                 with open(f'{self._output_path}/{augmentation_type}.tsv', 'w+') as f:
-                    zipped = zip(self._augmented_sentence_pairs[augmentation_type]['hun'],
-                                 self._augmented_sentence_pairs[augmentation_type]['eng'])
-                    for hun_sent, eng_sent in zipped:
-                        f.write(f'{hun_sent}\t{eng_sent}')
+                    zipped = zip(self._augmented_sentence_pairs[augmentation_type]['src'],
+                                 self._augmented_sentence_pairs[augmentation_type]['tgt'])
+                    for src_sent, tgt_sent in zipped:
+                        f.write(f'{src_sent}\t{tgt_sent}')
                         f.write('\n')
         else:
             for augmentation_type in augmentation_types:
                 os.mkdir(f'{self._output_path}/{augmentation_type}')
                 if self.save_original:
                     os.mkdir(f'{self._output_path}/original_{augmentation_type}')
-                for lang in ['hun', 'eng']:
+                for lang in ['src', 'tgt']:
```

Comment:
With this, the extensions will be 'sr' and 'tg', but I think we should use the real language codes for better readability later

Reply:
The output path usually already contains the actual language codes, I would prefer to keep `src` and `tgt`, because it will be easier to parse the folders if we work on multiple experiments in parallel.

Reply:
Then it's good, just please fix where the code cuts the last character from the extensions